tst/teststandard/opers/AutomorphismGroup.tst

```diff
@@ -39,6 +39,8 @@ gap> StructureDescription(H);
 #
 gap> g:=PerfectGroup(IsPermGroup,30720,1);;
 gap> h:=g^(1,153);;
+gap> AutomorphismGroup(g);;
+gap> AutomorphismGroup(h);;
```

Comment:
So, these just test that the computations do not run into an error? But we don't even test whether the result is `fail` or a group. Can't one perhaps at least output, say, the size of the resulting groups (if they are known), or test somehow else that the two computed groups are plausible? If that's not easily possible (e.g. computing the sizes takes too long), perhaps at least  this could be changed to `IsGroup(AutomorphismGroup(g));`?

Reply:
No. These are intended to pullout calculations of the subsequent `IsomorphismTest` earlier ,to reduce the chance of a timeout for the automated tests.

Reply:
@fingolfin I've replaced it with `MemoryUsage`.